CHANGES.rst

```diff
@@ -313,6 +313,9 @@ New Features
   - Add bolometric absolute and apparent magnitudes, ``M_bol`` and ``m_bol``.
     [#4262]
 
+  - Ensured that with ``spectral_density`` equivalency one could also convert
```

Comment:
No, just a silly mistake to have it under new features; now corrected (I also rebased so there is no new commit)